src/db.js

```
@@ -15,18 +14,13 @@ const {
   POSTGRES_DB,
 } = process.env;
 
-/*
-  All credentials setted to default values (exsept password - it is exapmle)
-  replace if needed with your own
-*/
-
 const sequelize = new Sequelize({
   database: POSTGRES_DB || 'postgres',
   username: POSTGRES_USER || 'postgres',
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '1815',
```

Avoid a hard-coded fallback password. `password: POSTGRES_PASSWORD || '1815'` embeds a secret in source and may hide misconfiguration when tests expect the env var to be provided. Prefer one of these approaches:

- Use `password: POSTGRES_PASSWORD` and fail early if it's missing (throw or log a clear error).
- Or keep a non-sensitive default but document it clearly. 

Also ensure `POSTGRES_PORT` is cast to a number (e.g. `port: Number(POSTGRES_PORT) || 5432`) so Sequelize receives the expected type.